src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -379,7 +597,6 @@ function isCtrlKey(e) {
 }
 pre.sf-dump .sf-dump-ellipsis {
     display: inline-block;
-    overflow: visible;
```

Comment:
I'm sure I borrow this line from some post on the net from someone would knew better css than me.
Since css is sometimes magic, does it hurt to keep it this way?